handler/handler.go

```diff
@@ -29,7 +30,11 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, cfg *config.Config
 	}
 	log.Info(ctx, "event handler called", logData)
 
-	contentPublished, err := h.ZebedeeCli.GetPublishedData(ctx, event.URL)
+	// Strip off data.json
+	uri := strings.TrimSuffix(event.URL, "/data.json")
+	log.Info(ctx, "event.URL strip off uri", log.Data{"uri": uri})
```

Comment:
I don't think this log line is required as its not providing valuable insight into the app functionality. Consider removing it